src/scripts/qa.ts

```diff
@@ -470,8 +478,8 @@ async function testRfqs(
       }
       const receivedAmountDecimals = new BigNumber(
         provided === 'base'
-          ? rfq.quoteData?.quoteTokenAmount ?? '0'
-          : rfq.quoteData?.baseTokenAmount ?? '0'
+          ? rfq?.quotes?.[0]?.quoteData?.quoteTokenAmount ?? '0'
+          : rfq?.quotes?.[0]?.quoteData?.baseTokenAmount ?? '0'
```

Comment:
The v2 API only supported one rfq at a time. V3 supports multiple. Should we update the qa test to send multiple rfqs per API call? For now I'm just sending one, so the response also only contains one.

Reply:
I think to minimize the changes let's leave it like this. Later we can modify.

Reply:
probably add a todo here to support multiple rfqs in one request - I think we do need to support it, since we were able to uncover issue with neptune with more than 1 rfqs in a call.